### DISCUSSION TITLE
GIthub Education Approved And Benefits not available yet

### DISCUSSION THREAD

**Discussion by salman-khan03** [community/community#139045] [GitHub Education]
### Hi everyone,

HI 

My name is Salman Khan. My GitHub ID is Salman-Khan3. I got my application approved for the GitHub education pack, but I am still waiting on the benefits. I would like you to please let me know as soon as my benefits are available. Do I need to get the GitHub subscription, which is $10$ per month, or is it free for students through the education pack? Please let me know. 

Thank You
Salman Khan

**Comment by davevad93** [comment]
Hi @Salman-Khan3 ,

If you mean the GitHub Pro and Copilot features, they are included in the `Student Developer Pack` for free.

Regarding your benefits, what's the status of your application?

If the label is green and states "`Congrats! Your request for @Salman-Khan3 was approved and your benefits will be available soon.` that means you have to wait. It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

If the label is purple and states "`Congrats! Your request for @Salman-Khan3 was approved and your benefits are now available.`", that means you should have access to your benefits. Try first going to the [pack offers page](https://education.github.com/pack) and select one of the available offers, if you can access them you are fine, if not I suggest you to raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.

For further reference, take a look at the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352):

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

And also the following discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721

**Reply by salman-khan03** [reply]
Thank you

**Reply by davevad93** [reply]
You're welcome @Salman-Khan3 👍

**Comment by matheuslpiresm** [comment]
I was also approved and I'm waiting for email, the last update was:

Congrats! Your request was approved and your benefits will be available!
